relay-server/src/actors/processor.rs

```diff
@@ -2316,13 +2324,131 @@ impl Service for EnvelopeProcessorService {
 
 #[cfg(test)]
 mod tests {
+    use std::str::FromStr;
+
     use chrono::{DateTime, TimeZone, Utc};
     use relay_general::pii::{DataScrubbingConfig, PiiConfig};
 
     use crate::{actors::project::ProjectConfig, extractors::RequestMeta};
 
     use super::*;
 
+    struct TestProcessSessionArguments<'a> {
+        item: Item,
+        received: DateTime<Utc>,
+        client: Option<&'a str>,
+        client_addr: Option<net::IpAddr>,
+        metrics_config: SessionMetricsConfig,
+        clock_drift_processor: ClockDriftProcessor,
+        extracted_metrics: Vec<Metric>,
+    }
+
+    impl<'a> TestProcessSessionArguments<'a> {
+        fn check_item_kept(&mut self) -> bool {
```

Comment:
```suggestion
        fn run_session_processor(&mut self) -> bool {
```

It would be better to put in the name what exactly it does. And it process sessions.